### PR TITLE
Trade Events: USTDA endpoint is empty

### PR DESCRIPTION
**Pull request by calumhalcrow**
[#85847236]

### CODE REVIEW

File: app/importers/trade_event/ustda_data.rb [hunk 1]
```diff
@@ -49,9 +49,12 @@ def import
...
       fh = open(@resource, 'r:UTF-8')
-      fh.readline # Remove 'Last Updated: ...'
-      content = fh.read.encode('UTF-16le', invalid: :replace, replace: '').encode('UTF-8')
-      doc = CSV.parse(content, headers: true, header_converters: :symbol, encoding: 'UTF-8')
+
+      raw = fh.read.encode('UTF-16le', invalid: :replace, replace: '').encode('UTF-8')
+      rows_as_strings = raw.split(/[\r\n]+/)[1..-1]  # Deal with the absence of newlines, throw away invalid "Last Updated..." first row.
+
+      doc = CSV.parse(rows_as_strings.join("\n"), headers: true, header_converters: :symbol, encoding: 'UTF-8')
+
```
Comment (calumhalcrow) [inline app/importers/trade_event/ustda_data.rb]: @carloslima @jamesisaacs any ideas on how to do this more...elegantly? See [#85847236](https://www.pivotaltracker.com/story/show/85847236).

Comment (carloslima) [inline app/importers/trade_event/ustda_data.rb]: I would expect this to solve the problem:

`content = fh.read.encode('UTF-16le', invalid: :replace, replace: '', :universal_newline => true).encode('UTF-8')`

Comment (calumhalcrow) [inline app/importers/trade_event/ustda_data.rb]: Looks much more elegant :-)

Comment (jamesisaacs) [inline app/importers/trade_event/ustda_data.rb]: Nice.

**Comment by calumhalcrow** [comment]
@carloslima with your suggestion, and using the "skip_lines" CSV option, we now have a much more elegant fix. Care to review/merge?

**Comment by carloslima** [comment]
I prefer the explicit `.readline` for throwing out the first line but either way is fine. :-)

Merging.

**Pull request merged**